contractcourt/htlc_timeout_resolver.go

```diff
@@ -378,6 +383,11 @@ func (h *htlcTimeoutResolver) spendHtlcOutput() (*chainntnfs.SpendDetail, error)
 		"fully confirmed", h, h.htlcResolution.ClaimOutpoint,
 		outpointToWatch)
 
+	// TODO(yy): add mempool watch here or in waitForSpend?
+	// if h.Mempool != nil {
+	// 	eventChan := h.Mempool.SubscribeMempoolSpent(outpointToWatch)
```

Comment:
This needs to be done in the link: this resolver is only active once that incoming channel hits the chain. At that point, we've missed the opportunity for the fast swap settle as we're dealing with a force close at the swap server.

Reply:
Think this doesn't need to be done in the link, this is for the outgoing channel hitting the chain. So this should ideally send back a resolution message

Reply:
> So this should ideally send back a resolution message

Ah true...the resolution message will be routed into the link anyway. Should be easy enough to exercsise in an itest.